### ISSUE TITLE
Generate and review validation reports: internet_archive - HOLD for CIC development

### ISSUE THREAD

**Issue by christinklez** [ucldc/rikolti#596]

**Comment by christinklez** [comment]
@aturner - let me know what you think! Some notes below.

## summary
Reviewed 2/2 collections

See notes in lines 351-352: https://docs.google.com/spreadsheets/d/1XkBwi8jiuGgrWvQBqjkrbavSl13yyrQNyRyEb_CQ4z0/edit?gid=1143782640#gid=1143782640

I think we should accept this mapper. I'm not seeing loss of data. Particularly in the BAMPFA collection, it seems we're mapping additional data (e.g., contributor, creator, rights)

The only thing I'm wondering is if we should *not* map "collection" (vernacular data) to "relation" (Calisphere). But I think it's fine to keep this.

## harvested and reviewed

**Theater Guest Recordings Collection**
https://calisphere.org/collections/27616/
https://calisphere-stage.cdlib.org/collections/27616/

**Heath (Brian & Edith)/Heath Ceramics Collection**
https://calisphere.org/collections/27369/
https://calisphere-stage.cdlib.org/collections/27369/

**Comment by aturner** [comment]
@christinklez agreed if we can remove the "collection"->"relation" mapping, that'd be great -- as that information doesn't appear to be displaying in the IA view.

It also looks like there's Description data that we didn't previously get -- it wasn't previously getting mapped, and I don't think should be a showstopper to moving forward. But if we are making any remaining adjustments, could we also get that data?

https://archive.org/details/bampfa-audio_00299
https://calisphere-stage.cdlib.org/item/8e1b04199bae742d7187d2e56cefda31/
https://calisphere.org/item/8e1b04199bae742d7187d2e56cefda31/

**Comment by christinklez** [comment]
@aturner - Barbara removed the "collection" -> "relation" mapping. Otherwise, it turns out the mapper was already giving us "description" - but only if that exists in the fetched data. Details here: https://github.com/ucldc/rikolti/issues/1317#issuecomment-4435111958

Okay to accept?

**Comment by aturner** [comment]
@christinklez I think so!

**Comment by christinklez** [comment]
Calling this accepted!
